checklist.md

```diff
@@ -3,7 +3,7 @@
 #### Don't begin class or method implementation with an empty line.  
 Remove all redundant empty lines, be careful :)
 #### If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field.
-If you create a formatter, make it a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).
+If you create a ormatfter, make it a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).
```

Comment:
undo this change
before each commit, review the files you are committing, to avoid unintended changes